.github/workflows/test.yml

```diff
@@ -35,7 +35,12 @@ jobs:
       run: |
         pip install -r requirements.txt
         pip install -r dev-requirements.txt
-        pip install -e .
+        pip install --upgrade pytest-rerunfailures
+        pip install -e 'git+https://github.com/ckan/ckanext-scheming.git@release-3.0.0#egg=ckanext-scheming[requirements]'
```

Comment:
You will need to install requirements from ckanext-scheming

Reply:
`ckanext-scheming` are added requirements to setup.py: https://github.com/ckan/ckanext-scheming/blob/master/setup.py so they are installed from there and you can check it in actions logs, e.g.: https://github.com/GenomicDataInfrastructure/gdi-userportal-ckanext-fairdatapoint/actions/runs/7447884366/job/20261062267#step:4:110

Reply:
Ah cool, I didn't know that. Thanks for sharing it.